TestHelpers.Tests/StringExtensionsTests.cs

```diff
@@ -56,5 +56,40 @@ public void SplitLines_InputWithOneLine_ShouldReturnOnlyOneLine()
             Assert.That(result, Is.EquivalentTo(expected));
         }
 
+        [Test]
+        public void CleanPath_DriveRoot_PreserveTrailingSlash()
+        {
+            Assert.AreEqual(@"c:\", @"c:\".CleanPath());
```

Comment:
It may seem a little silly / overly verbose, but I think I'd prefer if this was broken up into three lines for AAA (no comments for each section though).

You'll also need XFS.Path for Linux shenanigans.